resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+        {
+          "code": "BAD_REQUEST",
+          "message": "Enter lisaManagerReferenceNumber in the correct format, like Z1234"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <p>Request with a invalid payload</p>
+                        <p class="code--block">
+                            lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a>
+                        </p>
+                    </td>
+                    <td>
+        <pre class="code--block">
+        {
+          "accountId": "1234=5678"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">400 (Bad Request)</code></p>
+        <pre class="code--block">
+          {
+            "code": "BAD_REQUEST",
+            "message": "Bad Request",
+            "errors": [
+              {
+                "code": "INVALID_FORMAT",
+                "message": "Invalid format has been used",
+                "path": "/accountId"
+              }
+            ]
+          }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request for an account that is open or active</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "accountId": "2000000403"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">403 (Forbidden)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_ACCOUNT_ALREADY_OPEN",
+          "message": "The account already has a status of Open"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request for an account that is closed with a closure reason as transferred out</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "accountId": "0000000403"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">403 (Forbidden)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_ACCOUNT_ALREADY_CLOSED",
+          "message": "You cannot reinstate this account because it was closed with a closure reason of transferred out"
+        }
+        </pre>
+                    </td>
+                </tr>
+                 <tr>
+                    <td><p>Request for an account that is closed with a closure reason as cancelled</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
```

Comment:
Once the duplicated Testing section in this file is removed, please update link from`#section/Testing` to `/api-documentation/docs/api/service/lisa-api/2.0#testing`. Thanks